Entities/ACrab.h

```diff
@@ -690,6 +690,11 @@ ENTITYALLOCATION(ACrab)
 
 protected:
 
+	// Dynamic AimRange -- The maximum angle MountedMO can be aimed up, positive values only -- MaximDude
```

Comment:
See notes way above about spacing and comment style. Also, member variable changes are lower down and this should go somewhere in that area rather than here.